src/Buckle/Buckle.Tests/CodeAnalysis/EvaluatorTests.cs

```diff
@@ -777,7 +779,7 @@ expected identifier
     private void AssertValue(string text, object expectedValue) {
         var syntaxTree = SyntaxTree.Parse(text);
         var compilation = Compilation.CreateScript(null, syntaxTree);
-        var variables = new Dictionary<VariableSymbol, object>();
+        var variables = new Dictionary<VariableSymbol, EvaluatorObject>();
```

Comment:
To support references, the evaluator passes around EvaluatorObjects instead of normal Objects